awx/ui/client/src/templates/workflows/workflow-chart/workflow-chart.directive.js

```diff
@@ -646,7 +646,7 @@ export default ['moment', '$timeout', '$window', '$filter', 'TemplatesStrings',
                                 if(d.job.status === "successful"){
                                     return "#5cb85c";
                                 }
-                                else if (d.job.status === "failed" || d.job.status === "error" || d.job.status === "cancelled") {
+                                else if (d.job.status === "failed" || d.job.status === "error" || d.job.status === "canceled") {
```

Comment:
This is going to break jobs which have `cancelled` in the database.

Reply:
Yea, you've found a bug here as well:

https://github.com/ansible/awx/blob/devel/awx/main/models/unified_jobs.py#L74